### PR TITLE
feat: Provide e2e variable for Azure Event Hubs

### LINKED ISSUE #3720: Setup Azure infrastructure to introduce Azure Event Hubs for e2e tests
Setup Azure infrastructure to introduce Azure Event Hubs for e2e tests:

- Create new Azure Event Hubs namespace 
- Introduce `manage` connection string in `AZURE_EVENT_HUB_CONNECTION_STRING`

See https://github.com/kedacore/keda/pull/3717#issuecomment-1264666311 for context

### PR DESCRIPTION
**Pull request by tomkerkhove**
Provide e2e variable for Azure Event Hubs.

### Checklist

- [ ] When introducing a new scaler, I agree with the [scaling governance policy](https://github.com/kedacore/governance/blob/main/SCALERS.md)
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))
- [ ] Tests have been added
- [ ] A PR is opened to update our Helm chart ([repo](https://github.com/kedacore/charts)) *(if applicable, ie. when deployment manifests are modified)*
- [ ] A PR is opened to update the documentation on ([repo](https://github.com/kedacore/keda-docs)) *(if applicable)*
- [ ] Changelog has been updated and is aligned with our [changelog requirements](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#Changelog)

Closes #3720

### CODE REVIEW

File: .github/workflows/pr-e2e.yml [hunk 1]
```diff
@@ -141,6 +141,7 @@ jobs:
           AZURE_DEVOPS_PAT: ${{ secrets.AZURE_DEVOPS_PAT }}
           AZURE_DEVOPS_POOL_NAME: ${{ secrets.AZURE_DEVOPS_POOL_NAME }}
           AZURE_DEVOPS_PROJECT: ${{ secrets.AZURE_DEVOPS_PROJECT }}
+          AZURE_EVENT_HUB_CONNECTION_STRING: ${{ secrets.AZURE_EVENTHBUS_MANAGEMENT_CONNECTION_STRING }}
```
Comment (JorTurFer) [inline .github/workflows/pr-e2e.yml:144]: Let's name the env as the secret, for the maintenance will be easier. I'll rename it in my PR

File: .github/workflows/template-main-e2e-test.yml [hunk 2]
```diff
@@ -36,6 +36,7 @@ jobs:
           AZURE_DEVOPS_PAT: ${{ secrets.AZURE_DEVOPS_PAT }}
           AZURE_DEVOPS_POOL_NAME: ${{ secrets.AZURE_DEVOPS_POOL_NAME }}
           AZURE_DEVOPS_PROJECT: ${{ secrets.AZURE_DEVOPS_PROJECT }}
+          AZURE_EVENT_HUB_CONNECTION_STRING: ${{ secrets.AZURE_EVENTHBUS_MANAGEMENT_CONNECTION_STRING }}
```
Comment (JorTurFer) [inline .github/workflows/template-main-e2e-test.yml:39]: Let's name the env as the secret, for the maintenance will be easier. I'll rename it in my PR

File: .github/workflows/pr-e2e.yml (hunk 1 above)
Comment (tomkerkhove) [reply .github/workflows/pr-e2e.yml:144]:
```suggestion
          AZURE_EVENTHBUS_MANAGEMENT_CONNECTION_STRING : ${{ secrets.AZURE_EVENTHBUS_MANAGEMENT_CONNECTION_STRING }}
```

File: .github/workflows/template-main-e2e-test.yml (hunk 2 above)
Comment (tomkerkhove) [reply .github/workflows/template-main-e2e-test.yml:39]:
```suggestion
          AZURE_EVENTHBUS_MANAGEMENT_CONNECTION_STRING : ${{ secrets.AZURE_EVENTHBUS_MANAGEMENT_CONNECTION_STRING }}
```

**Pull request merged**